paper-toast.html

```diff
@@ -139,6 +168,14 @@
          * Overridden from `IronOverlayBehavior`.
          * Called when the value of `opened` changes.
          */
+
+        /**
+         * Fired when `paper-toast` is opened.
+         *
+         * @event 'iron-announce'
+         * @param {Object} detail
+         * @param {String} detail.text The text that will be announced.
+         */
```

Comment:
This block makes it look like the documentation is for `_openedChanged`. Usually we put event docs  towards the bottom of the implementation, for example: https://github.com/PolymerElements/paper-slider/blob/master/paper-slider.html#L739-L759